libconcurrent/bench.sh

```diff
@@ -3,12 +3,13 @@
 function usage()
 {
     echo -e "Usage: ./bench.sh FILE.run OPTION=NUM ...";
-    echo -e "This script runs the algorithm of FILE.run 10 times and calculates the average throughput (operations/second)";
+    echo -e "This script runs the algorithm of FILE.run multiple times for various number of threads and calculates the average throughput (operations/second)";
     echo -e " "
     echo -e "The following options are available."
-    echo -e "-t, --threads \t set the number of threads (fiber threads also included, if any) to be used in the benchmark"
+    echo -e "-s, --step \t set the step (extra number of threads to be used) in succesive set of iterations of the benchmark, default is number of processors/8 or 1"
+    echo -e "-t, --max_threads \t set the maximum number number of threads (fiber threads also included, if any) to be used in the last set of iterations of the benchmark, default is the number of system cores"
     echo -e "-f, --fibers  \t set the number of user-level threads per posix thread"
```

Comment:
Now, it seems that the '-f' option does not behave as expected. The issue is that the '-n' is not fixed.

Reply:
what exactly is '-n'?
Can you explain?

Reply:
I modified how the bench.sh and the executable parses arguments. Now "-t" is always the number of POSIX threads.